nilearn/tests/test_niimg_conversions.py

```diff
@@ -389,23 +389,88 @@ def test_iter_check_niimgs_memory():
 
 
 def test_repr_niimgs():
-    # Test with file path
+    # Tests with file path
     assert _utils._repr_niimgs("test") == "test"
+    assert _utils._repr_niimgs("test", shorten=False) == "test"
+    # Shortening long names by default
+    assert (_utils._repr_niimgs('/this/is/a/very/long/name/for/a/nifti/file') ==
+            '/this/is/a/very/lo...')
+    # Explicit shortening of long names
+    assert (_utils._repr_niimgs('/this/is/a/very/long/name/for/a/nifti/file',
+                                shorten=True) ==
+            '/this/is/a/very/lo...')
+    # Force long display of long names
+    assert (_utils._repr_niimgs('/this/is/a/very/long/name/for/a/nifti/file',
+                                shorten=False) ==
+            '/this/is/a/very/long/name/for/a/nifti/file')
+
```

Comment:
Could be rewritten more nicely by using 
`long_str='this/is/a/very/long/name/for/a/nifti/file'`
`short_str = '/this/is/a/very/lo...'`